.azure/pipelines/richnav.yml

```diff
@@ -0,0 +1,64 @@
+#
+# See https://docs.microsoft.com/en-us/vsts/pipelines/yaml-schema for details on this file.
+#
+
+# Configure which branches trigger builds
+trigger:
+  branches:
+    include:
+    - blazor-wasm
+    - master
+    - release/*
+    - internal/release/*
+
+variables:
+- name: _BuildArgs
+  value: '/p:SkipTestBuild=true'
```

Comment:
If this were left out, would the test code get rich nav too❔ Might be worthwhile.